### ISSUE TITLE
One selector - multiple elements

### ISSUE THREAD

**Issue by tommyno** [garris/BackstopJS#253]
Is it possible to use one selector, ex: `.container` to target and get screenshots of all matching elements? 

I have created a styleguide and would like to use BackstopJS to test all the different items. They are all wrapped in a ".container". Hope there is a simpler way than to write:

`.container:nth-of-type(1)`,
`.container:nth-of-type(2)`,
`.container:nth-of-type(3)`,
`...`
`.container:nth-of-type(42)`,

**Comment by nobso** [comment]
@tommyno  Is #249 related to your requirement?

**Comment by tommyno** [comment]
No, I don't think so. All of the `.container` wrapper divs has different content and need to create different reference screenshots. 

The linked issue (#249) is quite different: "to run tests on many similar selectors against one reference screenshot."

**Comment by nobso** [comment]
@tommyno i see. Thanks.

**Comment by garris** [comment]
@tommyno actually, I think #249 *is* exactly related to your issue.  There was an initial misunderstanding earlier in the conversation.   @nobso is currently working on it and doing a great job.  Feel free to review the thread and add color or comments if you have any.   I hope this can go out in the next release.   Cheers.
